app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Aux from '../../../hoc/auxillary';
+import logo from '../../../assets/images/algolia.svg';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const noMargin = css`
+  margin-bottom: 0;
+`;
+const LargeLogo = styled.img`
+  ${noMargin}
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 15px;
+  height: auto;
+  width: 10%;
+  @media (max-width: 480px) {
```

Comment:
shouldn't need a media query and width should be 100%

Reply:
Necessary css changes made